index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+        integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+  <title>Student Grades Table</title>
+</head>
+<body>
+  <header class="student-grade-table m-5">
+      <h2>Student Grade Table</h2>
+  </header>
```

Comment:
To clarify my previous indentation comment, this `<h2>` is indented 4 spaces from the `<header>`'s indent, while the rest of the document uses 2 spaces for each level of indenting.

Reply:
ok i'll fix it.thanks Tim!